storybook/webpack.config.js

```diff
@@ -1,26 +1,13 @@
-// you can use this file to add your custom webpack plugins, loaders and anything you like.
-// This is just the basic way to add addional webpack configurations.
-// For more information refer the docs: https://goo.gl/qPbSyX
+// load the default config generator.
+const genDefaultConfig = require('@kadira/storybook/dist/server/config/defaults/webpack.config.js')
```

Comment:
Oh, I just discovered that the errors I was getting earlier disappear if I return the config without creating a `const newConfig`. I'll prepare a new commit